README.md

```diff
@@ -13,7 +13,7 @@ The Ghaf [vulnerability reports](./reports/) available on this repository are au
 - [Ghaf 'main' generic-x86_64-release](./reports/main/packages.x86_64-linux.generic-x86_64-release.md)
 - [Ghaf 'main' riscv64-linux.microchip-icicle-kit-release](./reports/main/packages.riscv64-linux.microchip-icicle-kit-release.md)
 - [Ghaf 'ghaf-23.09' generic-x86_64-release](./reports/ghaf-23.09/packages.x86_64-linux.generic-x86_64-release.md)
-- [Ghaf 'ghaf-23.06' generic-x86_64-release](./reports/ghaf-23.06/packages.x86_64-linux.generic-x86_64-release.md)
+- [Ghaf 'ghaf-23.12' generic-x86_64-release](./reports/ghaf-23.12/packages.x86_64-linux.generic-x86_64-release.md)
```

Comment:
The commit removes ghaf-23.06 directory, but does not create ghaf-23.12 directory, with this .md file in it.

Reply:
Directory `ghaf-23.12` *should* be automatically generated by the github action, although, I'm not 100% sure if the `git-auto-commit-action` will will pick-up changes to completely new folders. 

We'll see in ~1.5h when this action is ready: https://github.com/tiiuae/ghafscan/actions/runs/7740903556

Reply:
yeah I just run the action manually to verify the reports  23.12

Reply:
yes  reports 23.12 automatically created! https://github.com/tiiuae/ghafscan/tree/main/reports/ghaf-23.12